### PR TITLE
Sidebar resize

### PR DESCRIPTION
**Pull request by teamrc**
Here is my proposal to fix issue #14 .

This fix allows sidebar to close and open correctly AFTER it has been resized.

### CODE REVIEW

File: src/jquery.sidebar.js [hunk 1]
```diff
@@ -99,7 +97,7 @@
          * */
         self.on("sidebar:open", function() {
             var properties = {};
-            properties[settings.side] = settings.range[1];
+            properties[settings.side] = 0;
```
Comment (IonicaBizau) [inline src/jquery.sidebar.js]: :+1:

File: src/jquery.sidebar.js [hunk 2]
```diff
@@ -90,8 +90,6 @@
 
         }, options);
 
-        // Override range
-        settings.range = settings.range[settings.side];
```
Comment (IonicaBizau) [inline src/jquery.sidebar.js]: You also could remove the `range` field from the defaults as well.

File: src/jquery.sidebar.js [hunk 3]
```diff
@@ -114,7 +112,15 @@
          * */
         self.on("sidebar:close", function(callback) {
             var properties = {};
-            properties[settings.side] = settings.range[0];
+            if (settings.side == "left" || settings.side == "right")
```
Comment (IonicaBizau) [inline src/jquery.sidebar.js]: Please use `===` instead of `==`. Also, put the `{` at the ending of this line.

File: src/jquery.sidebar.js (hunk 2 above)
Comment (teamrc) [inline src/jquery.sidebar.js]: Its your call. I don't know if you have any needs for settings.range.

File: src/jquery.sidebar.js (hunk 3 above)
Comment (teamrc) [inline src/jquery.sidebar.js]: feel free to change this. I don't know how to update my git pull request. maybe you can do it on your end?

Comment (gabipetrovay) [inline src/jquery.sidebar.js]: Pull requests are automagically updated when you push on the branch from where you created the pull request.

**Comment by lcristianiim** [comment]
Thats usefull.

File: src/jquery.sidebar.js (hunk 3 above)
Comment (IonicaBizau) [inline src/jquery.sidebar.js]: @teamrc As @gabipetrovay pointed out, just keep pushing in this branch and the PR will be updated. You can [edit the file on GitHub](https://github.com/teamrc/jQuery-sidebar/edit/sidebar-resize/src/jquery.sidebar.js/) and commit the fixes.

**Comment by IonicaBizau** [comment]
Landed in 3.1.0. :ship: :tada:  Thanks @teamrc! :cake:

**Pull request closed without merging**